store/postgres/src/deployment.rs

```diff
@@ -275,6 +276,21 @@ pub fn features(conn: &PgConnection, site: &Site) -> Result<BTreeSet<SubgraphFea
         .collect()
 }
 
+/// This migrates subgraphs that existed before the raw_yaml column was added.
+pub fn set_manifest_raw_yaml(
+    conn: &PgConnection,
+    site: &Site,
+    raw_yaml: &str,
+) -> Result<(), StoreError> {
+    use subgraph_manifest as sm;
+
+    update(sm::table.filter(sm::id.eq(site.id)))
```

Comment:
It would be better to also include `.filter(sm::raw_yaml.ne(raw_yaml))` to avoid unnecessary updates in that table. Otherwise, we'd rewrite the entire table on each node start.

Reply:
Sounds good! The sideeffect is that we'll never update the `raw_yaml` once it's set, but that shouldn't be an issue